libs/server/Resp/RespCommandAccessor.cs

```diff
@@ -1,16 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-namespace Garnet.server
-{
-    /// <summary>
-    /// RESP command accessor
-    /// </summary>
-    public static class RespCommandAccessor
```

Comment:
Cluster code is considered a separate plugin/project and we do not want it to access RespCommand directly. We might make the latter internal in future. That is why this accessor exists.